EpicSpawners-Plugin/src/main/java/com/craftaro/epicspawners/EpicSpawners.java

```diff
@@ -104,8 +104,15 @@ public void onPluginDisable() {
         if (!this.spawnerManager.wasConfigModified()) {
             this.saveToFile();
         }
-        this.particleTask.cancel();
-        this.spawnerCustomSpawnTask.cancel();
+
+        if (Bukkit.getScheduler().isCurrentlyRunning(this.particleTask.getTaskId())) {
```

Comment:
I'm not sure if `#isCurrentlyRunning` is the correct method here...
If I understand the [JavaDocs](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/scheduler/BukkitScheduler.html#isCurrentlyRunning(int)) for it correctly, for repeating-tasks it actually tells you if they are running right now and not when they are idle and waiting for their next scheduled execution

Have you experienced weird behavior that could be related to that? Is there an issue in canceling a task when it has already been cancelled in the past?
> A repeating task might not be running currently, but will be running in the future.